### PR TITLE
Community userpics

### LINKED ISSUE #160: Replace user pics with GitHub's versions
ATM the user pics for the OSS section are from http://uifaces.com (down at the moment of writing). We should replace those images with the real contributors' userpics from GitHub. 

Initially my idea was to fetch them from GitHub servers (using API). Not sure how this result in the overal loading performance tho. So this could be worth experimenting with.

### PR DESCRIPTION
**Pull request by okonet**
This should resolve #160 

Some notes: GitHub doesn't allow to request avatar images in specific size. Although I've implemented lazy loading and avatars will be loaded only when in viewport, they still generate some amount of traffic. There is just one way around it IMO: to pre-fetch data and generate this image on build step. In this case it can't be interactive (no hover effect) but we can stretch the background in this case.

I'm open for discussion on this one.

### CODE REVIEW

File: scripts/webpack.config.babel.js [hunk 1]
```diff
@@ -86,6 +86,9 @@ export default {
   ),
 
   plugins: [
+    new webpack.ProvidePlugin({
+      "fetch": "imports?this=>global!exports?global.fetch!whatwg-fetch",
```
Comment (thangngoc89) [inline scripts/webpack.config.babel.js]: fetch was imported by statinamic https://github.com/postcss/postcss.org/blob/master/scripts/index-client.js#L6

**Comment by mxstbr** [comment]
I think this is fine, prefetching in the build step doesn't massively improve the request size and is a lot more complex.

@ai @marcustisater what do you think?

**Comment by marcustisater** [comment]
Sorry, I'm still troubleshooting #178 

will get back asap once that's resolved for me. Did you get my gitter message @mxstbr ?

**Comment by marcustisater** [comment]
Great work @okonet, I agree with @mxstbr. 

LGTM

**Pull request merged**